### DISCUSSION TITLE
array_merge(): Expected parameter 3 to be an array, null given

### DISCUSSION THREAD

**Discussion by samehdoush** [livewire/livewire#3882] [7. Bugs (archived)]
when using  `` return redirect()->route('create_product'); ``

array_merge(): Expected parameter 3 to be an array, null given
![image](https://user-images.githubusercontent.com/48515888/134898271-7d654e85-1e98-4427-95d5-6a090ba89c7d.png)

``` 
 public function mount()
    {

   
        $this->user = auth()->user();
        if (!get_option('active_multi_vendor')) {
            return redirect()->route('home');
        }

        if (auth()->user()->hasRole('admin') || auth()->user()->hasRole('vendor')) {
    
            if (auth()->user()->status) {
               
                return redirect()->route('create_product');
                // dd('test');
            }
        }
}
````

livewire : 2.6.7
laravel : 8.61.0 
Before the livewire update it was working fine, is there any help?

**Comment by joshhanley** [comment]
@samehdoush skip render on redirect was removed in the last release, which impacts the redirect in mount.

So this functionality has been restored and will be in the next release 🙂
See PR #3971 for details.

Hope this helps!
